src/rpg_window/Window_Base.d.ts

```diff
@@ -0,0 +1,127 @@
+import { Window } from "../rpg_core/Window";
+import { Rectangle } from "../rpg_core/Rectangle";
+
+declare class TextState{
```

Comment:
It is better to declare it as an Interface and not a class since class implies it's usable with the prototype structure and keyword new. Which isn't the case it's an Object so 'interfaces' is better in this case.

Reply:
I added a new commit to switch it to interface.